pkg/controller/noobaa/noobaa_controller.go

```diff
@@ -78,7 +78,7 @@ func Add(mgr manager.Manager) error {
 	ownerHandler := &handler.EnqueueRequestForOwner{IsController: true, OwnerType: &nbv1.NooBaa{}}
 
 	err = c.Watch(&source.Kind{Type: &nbv1.NooBaa{}}, &handler.EnqueueRequestForObject{},
-		noobaaPredicate, &logEventsPredicate)
+		util.IgnoreIfNotInNamespace(options.Namespace), noobaaPredicate, &logEventsPredicate)
```

Comment:
noobaa package is the system itself and will be created under the namespace, I do understand that it is for making sure we are preserving the previous behaviour, but not sure we have to use it, WDYT?

Reply:
Yes, actually this is just preserving the previous behaviours. Theoretically, we can remove this and let NooBaa operator provision multiple systems in multiple namespaces however I do not understand the ramifications of doing so at the moment 😅 .